app/assets/javascripts/discourse/app/components/add-topic-status-classes.js

```diff
@@ -0,0 +1,49 @@
+import Component from "@ember/component";
+import { observes } from "discourse-common/utils/decorators";
+import { scheduleOnce } from "@ember/runloop";
+
+export default Component.extend({
+  didInsertElement() {
```

Comment:
should probably be a tagless component

Reply:
I still see the tagname

Reply:
I'm not seeing any tag name in my dev environment. I removed it [here](https://github.com/discourse/discourse/pull/12729/files#diff-b9a444c238bd9d77d0b9357fd515533b51e3d136e8301cbb73339cb41f0c3a91R5).